api/rest-api-filters/rest-api-filters.php

```diff
@@ -0,0 +1,109 @@
+<?php
+
+register_woocommerce_admin_test_helper_rest_route(
+	'/rest-api-filters',
+	[ WCA_Test_Helper_Rest_Api_Filters::class, 'create' ],
+	array(
+		'methods' => 'POST',
+		'args'                => array(
+			'endpoint'     => array(
+				'description'       => 'Rest API endpoint.',
+				'type'              => 'string',
+				'required'			=> true,
+				'sanitize_callback' => 'sanitize_text_field',
+			),
+			'dot_notation' => array(
+				'description'       => 'Dot notation of the target field.',
+				'type'              => 'string',
+				'required'			=> true,
+				'sanitize_callback' => 'sanitize_text_field',
+			),
+			'replacement'   => array(
+				'description'       => 'Replacement value for the target field.',
+				'type'              => 'string',
+				'required'			=> true,
+				'sanitize_callback' => 'sanitize_text_field',
+			),
+		),
+	)
+);
+
+register_woocommerce_admin_test_helper_rest_route(
+	'/rest-api-filters',
+	[ WCA_Test_Helper_Rest_Api_Filters::class, 'delete' ],
+	array(
+		'methods' => 'DELETE',
+		'args'                => array(
+			'index'     => array(
+				'description'       => 'Rest API endpoint.',
+				'type'              => 'integer',
+				'required' => true,
+			),
+		),
+	)
+);
+
+
+register_woocommerce_admin_test_helper_rest_route(
+	'/rest-api-filters/(?P<index>\d+)/toggle',
+	[ WCA_Test_Helper_Rest_Api_Filters::class, 'toggle' ],
+	array(
+		'methods' => 'POST',
+	)
+);
+
+
+class WCA_Test_Helper_Rest_Api_Filters {
+	const WC_ADMIN_TEST_HELPER_REST_API_FILTER_OPTION = 'wc-admin-test-helper-rest-api-filters';
+
+	public static function create( $request ) {
+		$endpoint = $request->get_param( 'endpoint' );
+		$dot_notation = $request->get_param( 'dot_notation' );
+		$replacement = $request->get_param( 'replacement' );
+
+		if ( $replacement === 'false' ) {
+			$replacement = false;
```

Comment:
It seems that we could use `filter_var( $replacement, FILTER_VALIDATE_BOOLEAN );`?

https://stackoverflow.com/questions/4775294/parsing-a-string-into-a-boolean-value-in-php

Reply:
Nice! I did not know about that :) 

Fixed in https://github.com/woocommerce/woocommerce-admin-test-helper/pull/37/commits/f0e8ad9c86a4ee5c40e081f9e27c86c05bf50f38